seedfarmer/commands/_parameter_commands.py

```diff
@@ -36,6 +36,13 @@ def generate_export_env_params(metadata: Optional[Dict[str, Any]]) -> Optional[L
     return envs
 
 
+def generate_export_raw_env_params(metadata: Optional[Dict[str, Any]]) -> Optional[List[str]]:
+    envs: List[str] = []
+    if metadata is not None and metadata.keys() is not None:
```

Comment:
`metadata.keys()` will never return `None` even if the metadata dict is empty..

Reply:
removed